### DISCUSSION TITLE
What pain points does Linearmouse solve?

### DISCUSSION THREAD

**Discussion by kwokpia1** [linearmouse/linearmouse#499] [General]
No offense intended, what is the necessity of using Linearmouse instead of Option+, or what pain points does Linearmouse solve?

**Comment by lujjjh** [comment]
Well, it's pretty simple really - there are plenty of mouse manufacturers out there besides Logitech. And even though I'm currently using a Logitech product, I find that Logi Options+ doesn't quite meet my needs, particularly when it comes to pointer and scrolling acceleration.

**Comment by eehmo** [comment]
I'd say, the main painpoint it solves for me is the incredibly dumb default macOS scroll direction behavior - by default either my trackpad or my mouse has its scrolling reversed. Linearmouse lets me change the direction for one of them, so my mouse scrolls normally and I keep natural scrolling for my trackpad.

**Comment by hanleybrand** [comment]
The main pain point is solved for me is that Logitech stopped supporting the G300s mouse _before I picked one up at Target_ -- that's right, they gave up on supporting a device that is still for sale on newer hardware/operating systems. 

I don't have a ton of experience with logitech, but the little experience I've had is that options+ seems fine for the wireless keyboard/mouse combo my work bought for me but for some reason they didn't see the point in also supporting wired devices, like the G300s mouse that seemed worth trying when I bought it like a year or so ago. 

Unfortunately, their wired-device-engineers apparently can't figure out how to support more recent versions of MacOS or the M1/M2 chips, and can't be bothered to talk to the wireless deivce engineers.

So in case this is coming across as vague or incoherent, the actual main pain point that linear mouse solves is that logitch sucks.

**Comment by j5shi** [comment]
Well, despite the physical size and shape of mouses, the most comfortable setting for a mouse user , e.g. acceleration, sensitivity, scroll speed, varies from person to person. And softwares like the LinearMouse provided a very nice solution to customize/fine-tuning those settings, which, typically can not be satisfied by operation systems.
